File: swagger/models/get_programs_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetProgramsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		programs
		Contains information about the programs.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Programs': 'programs',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'programs': 'Programs',
		}

		'PaginationResponse': ('pagination_response', PaginationResponse),
	programs = None

File: swagger/Models/GetProgramsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetProgramsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the programs.
	 * 
	 * @var Program[] 
	 */
	public $programs;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Programs' => ['programs', 'array', Program::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'programs' => ['Programs', 'array', Program::class],
			];
	}
}
